lib/team_api/joiner.rb

```diff
@@ -101,37 +108,20 @@ def initialize(site)
       @public_mode = site.config['public']
     end
 
+    # Jekyll seems to be removing non-alpha characters from the team member
+    # names, causing the TeamIndexer to not be able to find any team member
+    # matches. This changes all of the team member keys back to what we expect.
+    def restructure_team_data!
+      site.data['team'].keys.each do |key|
+        site.data['team'][site.data['team'][key]['name']] = site.data['team'].delete(key)
+      end unless site.data['team'].nil? || !site.data['team'].respond_to?(:keys)
```

Comment:
Is this doing the whole .each do/end block _unless_ the data isn't there? I think it would be much clearer to flip the logic to the more traditional:
```
if site.data['team'].present? && site.data['team'].respond_to?(:keys)
  site.data['team'].keys.each do |key|
   site.data['team'][site.data['team'][key]['name']] = site.data['team'].delete(key)
  end
end
```

(double check the switch there from the `unless` to the `if` logic, that's always tricky)